### PR TITLE
Align label and data on show page

### PR DESCRIPTION
**Pull request by davi-tapajos-codeminer42**
Issue: https://github.com/thoughtbot/administrate/issues/1213

Captura de Tela 2020-01-10 às 17 29 42

### CODE REVIEW

File: app/assets/stylesheets/administrate/components/_cells.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .cell-label {
+  padding-top: $top-space;
+
```
Comment (pablobm) [inline app/assets/stylesheets/administrate/components/_cells.scss]: Can you provide an example (screenshot) of how cell labels should also receive this padding? I can't recall from memory right now 😓

File: app/assets/stylesheets/administrate/library/_variables.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@ $heading-line-height: 1.2 !default;
 $base-border-radius: 4px !default;
 $base-spacing: $base-line-height * 1em !default;
 $small-spacing: $base-spacing / 2 !default;
+$top-space: 0.39em !default;
```
Comment (pablobm) [inline app/assets/stylesheets/administrate/library/_variables.scss]: I feel that `top-space` it a bit generic a name. Should this be something else? I guess it'll depend on whether `cell-label` needs to be altered (see my comment above).

File: app/assets/stylesheets/administrate/components/_cells.scss (hunk 1 above)
Comment (davi-tapajos-codeminer42) [reply app/assets/stylesheets/administrate/components/_cells.scss]: Sure

cell-label

Comment (pablobm) [reply app/assets/stylesheets/administrate/components/_cells.scss]: Thank you, I get it now :+1:

**Comment by pablobm** [comment]
OK @davi-tapajos-codeminer42, let's see if we can see this through!

I think an opinion from a designer may take a bit to arrive, so let's see what we can fix for now. What do you think of my comment about baseline vs centre alignment?

Also, I think the name `$top-space` is not very descriptive. Also it may be that we don't use the same measure in both places in the end, depending on how the baseline vs centre alignment question is resolved. What do you think?

**Comment by davi-tapajos-codeminer42** [comment]
I think that centre alignment will looks nice! 
and yep, probably we will get rid of this $top-space :)

**Comment by pablobm** [comment]
Cool. Would you be able to make those changes, and I'll review again? Thank you!

**Comment by davi-tapajos-codeminer42** [comment]
It is kind hard to align with "dt" and "dd" tags
Maybe in the future we could change those tags and use vertical-align to get better results

Captura de Tela 2020-01-23 às 11 49 51

**Comment by pablobm** [comment]
I think this is good enough for now. Thank you again @davi-tapajos-codeminer42 

Fixes https://github.com/thoughtbot/administrate/issues/1213

**Pull request merged**